### PR TITLE
fix several misleading ddocs and return types in the function isXXX()

### PR DESCRIPTION
**Pull request by BBasile**
their results are always of type  bool.

see previous failed PR (#3286)

### CODE REVIEW

File: std/math.d [hunk 1]
```diff
@@ -4589,7 +4589,7 @@ unittest
 
 
 /*********************************
- * Returns !=0 if e is a NaN.
+ * Returns $(D_KEYWORD true) if x is a NaN, and $(D_KEYWORD false) otherwise.
```
Comment (WalterBright) [inline std/math.d]: Use proper Ddoc format:
```
Determines if x is a NaN or not.
Params:
  x = a floating point value
Returns:
  true if x is a NaN
```

Comment (WalterBright) [inline std/math.d]: ditto for the rest

**Comment by andralex** [comment]
also just use D instead of D_KEYWORD

File: std/math.d (hunk 1 above)
Comment (WalterBright) [inline std/math.d]: also, saying `and false otherwise` is redundant, as there is no other possible value.

**Comment by BBasile** [comment]
what's wrong with [farm3](https://auto-tester.puremagic.com/show-run.ghtml?projectid=1&runid=1554111&isPull=true) ? it seems to fail with _druntime_, which is totally unrelated to this ridiculous PR !

**Pull request merged**